location/templates/apartment.html

```diff
@@ -2,7 +2,24 @@
 {% load crispy_forms_tags %}
 
 {% block script %}
-<link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.1.1/css/mdb.min.css" rel="stylesheet">
+<style>
```

Comment:
Since the CSS here and in account.html looks the same, is there a reason that we're not putting these in `css/styles.css` or in another stylesheet?

Reply:
Good idea! I was initially worried that it might interfere with the styling of other pages since I was importing a stylesheet but now since we are using a few classes for styling, I think it's safe to include it in the `css/styles.css` file. Moved the styles from `apartment.html` and `account.html` to `styles.css`.